twine/commands/register.py

```diff
@@ -73,7 +86,8 @@ def main(args):
         default="pypi",
         help="The repository to register the package to. Can be a section in "
              "the config file or a full URL to the repository (default: "
-             "%(default)s)",
+             "%(default)s). (Can also be set via %(env)s environment "
```

Comment:
These notes about environment variables should be split out into a wholly separate pull request.